### ISSUE TITLE
Update environment Wiki with Mujoco environments

### ISSUE THREAD

**Issue by zuoxingdong** [openai/gym#729]
It could be better to update the Wiki for environment with Mujoco ones, e.g. Swimmer etc. 

In addition, maybe better to have explanation for each dimension of action space, e.g. first dimension is position, second dimension is velocity.

**Comment by madvn** [comment]
I second this request for MuJoCo envs - see #106

**Comment by Amritpal-001** [comment]
@zuoxingdong  I compiled documentation on Mujuco's **Fetch environments** It doesn't cover  Swimmer, but Most of the principles are common amongst mujoco environment, so this might be helpful. I have tried to cover the observation and action variables, reward function, and comparison among all 4 fetch environments.  [https://link.medium.com/CV6la7YfV7](https://link.medium.com/CV6la7YfV7)

 I hope it's helpful.
